lib/howl/interactions/location_selection.moon

```diff
@@ -0,0 +1,46 @@
+-- Copyright 2012-2015 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+import app, interact, mode, Buffer from howl
+import highlight from howl.ui
+import Matcher from howl.util
+
+get_buffer_for_file = (file, local_buffers) ->
```

Comment:
Just a note: We could possibly have `local_buffers` be a table keyed by file path. That would cut down on the iteration required for revisited files.